dashboard/db/schema.rb

```diff
@@ -742,6 +742,8 @@
     t.integer  "script_id",       limit: 4
     t.integer  "level_source_id", limit: 4
     t.boolean  "submitted"
+    t.boolean  "view_answers"
```

Comment:
tbh, I'm starting to grow a little bit worried about trying to convert this all to use an enum. As I try to do so, I notice there are a bunch of places where we get/set `submitted` right now, and I'm concerned I'll end up regressing something.

Given that both approaches have their drawbacks, I'm somewhat inclined towards sticking with the current approach which has the benefit of having been tested by my for the past couple weeks while working on this stuff. I'll still rename the column (as I go back I think I like Andrew's suggestion of `readonly_answers` best).

If in the future we decide we really don't like this, we still have the option of doing another (admittedly time-consuming) migration.